ios/PurchasesHybridCommon/PurchasesHybridCommon/CommonFunctionality.swift

```diff
@@ -199,6 +199,41 @@ import RevenueCat
 
 }
 
+// MARK: In app messages
+@objc public extension CommonFunctionality {
+
+#if os(iOS) || targetEnvironment(macCatalyst) || VISION_OS
+    @available(iOS 16.0, *)
+    @available(tvOS, unavailable)
+    @available(macOS, unavailable)
+    @available(watchOS, unavailable)
+    @objc(showStoreMessagesCompletion:)
+    static func showStoreMessages(completion: @escaping () -> Void) {
+        _ = Task<Void, Never> {
```

Comment:
This is temporary. Planning to use completion block methods in iOS, but didn't expose it in the native SDK.